docs/mongocli/command/mongocli-cloud-manager-logs-keyProviders-list.txt

```diff
@@ -0,0 +1,71 @@
+.. _mongocli-cloud-manager-logs-keyProviders-list:
+
+=============================================
+mongocli cloud-manager logs keyProviders list
+=============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Lists all key provider configurations found in the encrypted log file.
+
+Syntax
+------
+
+.. code-block::
+
+   mongocli cloud-manager logs keyProviders list [options]
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -f, --file
+     - string
+     - true
+     - Path to the file that contains encrypted audit logs.
+   * - -h, --help
+     - 
+     - false
+     - help for list
+   * - -o, --output
+     - string
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file.
+
+Inherited Options
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Profile to use from your configuration file.
+
+Examples
+--------
+
+.. code-block::
+
+
+   $ mongocli ops-manager listKeyProvider --file log.gz
```

Comment:
Nice to have: do you have an example output? It would be useful to show this.

Reply:
I didn't find any command with an example output, do you have a suggestion on how this should be formatted? (below is a sample output, but probably some additional wording should be included).
Also, since we don't show example output for other commands, I'd say that we should not do this here either, for having a consistent description of the commands. What do you think?

```
kmip: UniqueKeyID = mdbKEK, KMIPServerName = 127.0.0.1, KMIPPort: 8081, KeyWrapMethod = get
```

Reply:
I think it is useful to show, in this case. As for how to format it, I would format it as the example itself, and add a comment "Here is an example of the output:"